tests/AutoReview/ProjectCodeTest.php

```diff
@@ -65,6 +65,7 @@
         \PhpCsFixer\Runner\FileFilterIterator::class,
         \PhpCsFixer\Runner\FileLintingIterator::class,
         \PhpCsFixer\StdinFileInfo::class,
+        \PhpCsFixer\Test\Assert\AssertTokensTrait::class,
```

Comment:
testing the public methods of a trait directly by using it is better than testing it through some usage, I see no diff. with testing public methods of a class directly by creating an instance of the class rather than testing the class through some other class.